docs/source/project_documentation/dev_guide/debugging_aac.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: default
+title: Debugging AaC
+parent: Developer's Guide to AaC
+nav_order: 1
+---
+
+# Debugging AaC In Gitpod
+
+The AaC repository is pre-configured for immediate development in Gitpod environments leveraging VSCode, but these steps will work in local environments as well.
+
+## Opening AaC In Gitpod
+
+There are several ways to open the repository in Gitpod, the easiest is to click the Gitpod 'Ready-to-Code' button in the readme.
+![Gitpod 'Ready-to-Code' button in the repository readme](../../../assets/images/debugging/gitpod_readme_button.png)
```

Comment:
Image reference pathing needs updated. Remember you can check within GitPod for a "local" launch of the documentation to confirm pathing for images and linkage between pages.